src/control/server/init/setup_spdk.sh

```diff
@@ -97,11 +97,9 @@ else
 		'/sys/class/uio/uio*/device/config'	\
 		'/sys/class/uio/uio*/device/resource*'; do
 
-		# shellcheck disable=SC2086
-		if list=$(ls -d $glob); then
-			echo -n "RUN: ls -d $glob | xargs -r chown -R "
-			echo "$_TARGET_USER"
-			echo "$list" | xargs -r chown -R "$_TARGET_USER"
+		if compgen -G "${glob}" > /dev/null; then
+			echo "RUN: chown -R ${_TARGET_USER} ${glob}"
+			chown -R "${_TARGET_USER}" ${glob}
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]

Reply:
Ugly workaround to this lint issue
```suggestion
			chown -R "${_TARGET_USER}" "$(compgen -G "${glob}")"
```